user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/IGQIOnDestroy.md

```diff
@@ -1,11 +1,22 @@
 ---
-uid: IGQIOnDestroy
+uid: GQI_IGQIOnDestroy
 ---
 
-# IGQIOnDestroy
+# IGQIOnDestroy Interface
 
-The *IGQIOnDestroy* interface is called when the instance object is destroyed, which happens when the session is closed, e.g. in case of inactivity or when all the necessary data has been retrieved. It can for instance be used to end the connection with a database. This interface has one method:
+## Definition
+
+Namespace: `Skyline.DataMiner.Analytics.GenericInterface`  
+Assembly: `SLAnalyticsTypes.dll`
+
+The *IGQIOnDestroy* interface is called when the instance object is destroyed, which happens when the session is closed, e.g. in case of inactivity or when all the necessary data has been retrieved. It can for instance be used to end the connection with a database.
```

Comment:
An interface can't really be "called".

```suggestion
The *IGQIOnDestroy* interface can be implemented for an Ad Hoc data source or Custom Operator to clean up resources after is has been used. It can for instance be used to close a database connection.
```

I would list all the scenarios in which the data source/operator can be destroyed in a separate section.